src/components/editor/property/InputLiteral.jsx

```diff
@@ -96,24 +96,31 @@ export class InputLiteral extends Component {
       const itemId = obj.id || shortid.generate()
 
       return <div id="userInput" key = {itemId} >
-        {obj.content}
-        <button
-          id="deleteItem"
-          type="button"
-          onClick={this.handleDeleteClick}
-          key={`delete${obj.id}`}
-          data-item={itemId}
-          data-label={this.props.formData.uri}
-        >X
-        </button>
+        <div
+          className="rbt-token rbt-token-removeable">
+          {obj.content}
+          <button
+            id={`delete${obj.id}`}
+            type="button"
+            onClick={this.handleDeleteClick}
+            key={`delete${obj.id}`}
+            data-item={obj.id}
+            data-label={this.props.formData.uri}
+            className="close rbt-close rbt-token-remove-button">
+            <span
+                aria-hidden="true"
+                data-item={obj.id}>×</span>
```

Comment:
is this needed here too?

Reply:
Yes, because depending on where you click, you either get the span or the button passed into the event.

Reply:
Why did you move away from `itemId`?

Reply:
Clearer. `const itemId = obj.id || shortid.generate()`. If, for some reason, obj.id were missing, then a generated shortid would be meaningless for deleting. So what you really need is obj.id.